targets/targets.json

```diff
@@ -4132,6 +4161,14 @@
         "supported_form_factors": ["ARDUINO"],
         "release_versions": ["5"],
         "device_has_remove": [],
+        "extra_labels_add": ["PSA"],
+        "components_add": ["FLASHIAP"],
+        "macros_add": [
+            "MBEDTLS_PSA_CRYPTO_C",
+            "MBEDTLS_ENTROPY_NV_SEED",
```

Comment:
This device has TRNG enabling nvseed is redundant

Reply:
Pelion require injection API regardless TRNG